In [39]:
# Define where you are running the code: colab or local
RunInColab          = False     # (False: no  | True: yes)

# If running in colab:
if RunInColab:
    # Mount your google drive in google colab
    from google.colab import drive
    drive.mount('/content/drive')

    # Find location
    #!pwd
    #!ls
    #!ls "/content/drive/My Drive/Colab Notebooks/MachineLearningWithPython/"

    # Define path del proyecto
    Ruta            = "/content/drive/My Drive/Colab Notebooks/MachineLearningWithPython/"

else:
    # Define path del proyecto
    Ruta            = "C:/Users/GeekD/OneDrive/Documents/GitHub/TC1002S/NotebooksProfessor"
    # Dataset url
    URL = Ruta + "/datasets/iris/iris.csv"

In [52]:
# Import the packages that we will be using
import pandas as pd

# Dataset url
URL = Ruta + "/datasets/iris/iris.csv"

# Load the dataset
df = pd.read_csv(URL, header = None)

In [53]:
# Print the dataset
df

0    1    2    3               4
0    5.1  3.5  1.4  0.2     Iris-setosa
1    4.9  3.0  1.4  0.2     Iris-setosa
2    4.7  3.2  1.3  0.2     Iris-setosa
3    4.6  3.1  1.5  0.2     Iris-setosa
4    5.0  3.6  1.4  0.2     Iris-setosa
..   ...  ...  ...  ...             ...
145  6.7  3.0  5.2  2.3  Iris-virginica
146  6.3  2.5  5.0  1.9  Iris-virginica
147  6.5  3.0  5.2  2.0  Iris-virginica
148  6.2  3.4  5.4  2.3  Iris-virginica
149  5.9  3.0  5.1  1.8  Iris-virginica

[150 rows x 5 columns]

In [26]:
#Rows
rows = df.shape[0]
rows

149

In [29]:
#Columns
cols = df.shape[1]
cols

5

In [56]:
#Data analysis
df[4].value_counts()


Iris-setosa        50
Iris-versicolor    50
Iris-virginica     50
Name: 4, dtype: int64

Load the iris.csv file in your computer and understand the dataset

How many observations (rows) are in total?
Hay 149 entradas.

How many variables (columns) are in total? What do they represent?
Se evalúan 5 variables

How many observations are for each type of flower?
Hay 50 de cada tipo de flor.

What is the type of data for each variable?
Int 64 bits

What are the units of each variable?
Enteros
